TYPO3.Flow/Classes/TYPO3/Flow/Object/ObjectSerializer.php

```diff
@@ -165,15 +169,15 @@ public function serializeObjectAsPropertyArray($object, $isTopLevelItem = true)
             } elseif (is_object($propertyValue)
                         && $this->persistenceManager->isNewObject($propertyValue) === false
                         && (
-                            $this->reflectionService->isClassAnnotatedWith($propertyClassName, 'TYPO3\Flow\Annotations\Entity')
-                            || $this->reflectionService->isClassAnnotatedWith($propertyClassName, 'TYPO3\Flow\Annotations\ValueObject')
-                            || $this->reflectionService->isClassAnnotatedWith($propertyClassName, 'Doctrine\ORM\Mapping\Entity')
+                            $this->reflectionService->isClassAnnotatedWith($propertyClassName, Flow\Entity::class)
+                            || $this->reflectionService->isClassAnnotatedWith($propertyClassName, Flow\ValueObject::class)
+                            || $this->reflectionService->isClassAnnotatedWith($propertyClassName, ORM\Entity::class)
                         )) {
                 $propertyArray[$propertyName][self::TYPE] = 'persistenceObject';
                 $propertyArray[$propertyName][self::VALUE] = get_class($propertyValue) . ':' . $this->persistenceManager->getIdentifierByObject($propertyValue);
             } elseif (is_object($propertyValue)) {
                 $propertyObjectName = $this->objectManager->getObjectNameByClassName($propertyClassName);
-                if ($this->objectManager->getScope($propertyObjectName) === \TYPO3\Flow\Object\Configuration\Configuration::SCOPE_SINGLETON) {
+                if ($this->objectManager->getScope($propertyObjectName) === Configuration\Configuration::SCOPE_SINGLETON) {
```

Comment:
I wanted to avoid the use statement here. But if you both think it would better that way ..